src/praisonai-agents/tests/unit/test_push_protocols.py

```
@@ -0,0 +1,447 @@
+        assert cfg.delivery.enabled is True
+        assert cfg.polling.enabled is True
+
+    def test_to_dict(self):
+        cfg = PushConfig(enabled=True, redis=RedisConfig())
+        d = cfg.to_dict()
+        assert d["enabled"] is True
+        assert d["redis"] is not None
+        assert d["redis"]["host"] == "localhost"
+
+    def test_redis_config_hides_secrets(self):
+        cfg = RedisConfig(password="secret123", url="redis://user:pass@host")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)